packages/runtime/render/src/server/rsc/rsc.tsx

```diff
@@ -0,0 +1,69 @@
+export { renderToReadableStream } from 'react-server-dom-webpack/server.edge';
+import type { ClientManifest } from '@modern-js/types/server';
+import { renderToReadableStream } from 'react-server-dom-webpack/server.edge';
+import { decodeReply } from 'react-server-dom-webpack/server.edge';
+export { createFromReadableStream } from 'react-server-dom-webpack/client.edge';
+export {
+  registerClientReference,
+  registerServerReference,
+} from 'react-server-dom-webpack/server';
+
+declare const __webpack_require__: (path: string) => any;
+
+type RenderRscOptions = {
+  element: React.ReactElement;
+  clientManifest: ClientManifest;
+  returnValue?: unknown;
+};
+
+export const renderRsc = (options: RenderRscOptions) => {
+  const payload = options.returnValue
+    ? { returnValue: options.returnValue, root: options.element }
```

Comment:
Where will returnValue be passed in?

Reply:
This is needed if we want to support re-rendering after sending an  server action, I've removed it.